internal/testing/cluster_test.go

```diff
@@ -644,3 +644,100 @@ func TestParallelUpload(t *testing.T) {
 	}
 	wg.Wait()
 }
+
+// TestEphemeralAccountSync verifies that setting the requiresSync flag makes
+// the autopilot resync the balance between renter and host.
+func TestEphemeralAccountSync(t *testing.T) {
+	if testing.Short() {
+		t.SkipNow()
+	}
+
+	dir := t.TempDir()
+	cluster, err := newTestCluster(dir, zap.NewNop())
+	if err != nil {
+		t.Fatal(err)
+	}
+	w := cluster.Worker
+
+	// add host
+	_, err = cluster.AddHosts(1)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	// Wait for account to appear.
```

Comment:
I added `WaitForAccounts` to the cluster

Reply:
Doesn't appear to be in master yet